github/ci/prow/files/config.yaml

```diff
@@ -98,6 +98,7 @@ tide:
     kubevirt/csi-driver: squash
     kubevirt/ssp-operator: squash
     kubevirt/must-gather: squash
+    kubevirt/user-guide: merge
```

Comment:
I figure this stands in conflict with the rules applied here: https://github.com/kubevirt/project-infra/blob/master/github/ci/prow/files/config.yaml#L292

See point 5 paragraph from [best practices](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/maintainers.md#best-practices) for how to handle the situation here.